Remotion/Validation/Core/Implementation/DiagnosticInformationValidatorFormatterDecorator.cs

```diff
@@ -62,6 +62,12 @@ public string Format (IPropertyValidator validator, Func<Type, string> typeNameF
         case ILengthValidator lengthValidator:
           return FormatLengthValidator(lengthValidator, typeName);
 
+        case IMinLengthValidator minLengthValidator:
+          return FormatMinimumLengthValidator(minLengthValidator, typeName);
+
+        case IMaxLengthValidator maxLengthValidator:
+          return FormatMaximumLengthValidator(maxLengthValidator, typeName);
+
```

Comment:
Given the updated setup, we have dedicated calls for Length, MaxLength, MinLength and ExactLength classes, not the intefaces. otherwise, we would log LengthValidator and ExactLengthValidator twice since we do not want the ILengthValidator interface.